docs/source/library_reference_c.rst

```diff
@@ -12,3 +12,11 @@ The interaction between Python and GPI is via a Python extension module called `
 traversing the hierarchy, getting/setting an object's value, registering callbacks etc.
 
 .. doxygenfile:: gpi.h
+   :sections: brief detaileddescription define typedef enum
```

Comment:
I had to set `:sections:` manually to exclude functions. Local docs build was excluding the functions because they are in doxygen groups, but readthedocs was showing them.